.github/workflows/repolinter.yml

```diff
@@ -6,8 +6,11 @@ name: Repolinter Action
 # Currently there is no elegant way to specify the default
 # branch in the event filtering, so branches are instead
 # filtered in the "Test Default Branch" step.
-on: [push, workflow_dispatch]
-
+on: 
+  push: 
+    paths-ignore:
+      - ".github/**" # skip changes to the .github branch (workflows, etc.)
```

Comment:
I'm not sure if this is something that we are always required to run.

Reply:
Shouldn't be an issue - the [rules set that repolinter](https://raw.githubusercontent.com/newrelic/.github/main/repolinter-rulesets/community-plus.yml) is using only checks for existence and required content in license files, etc.